portieris.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: portieris
+  version: 0.13.19
+  epoch: 0
+  description: A Kubernetes Admission Controller for verifying image trust.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/IBM/portieris.git
+      tag: v${{package.version}}
+      expected-commit: d0705039d060ab11b8fa598b79647ac0008af57e
+
+  - runs: |
+      VERSION=v${{package.version}} make portieris
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin/
+      install -m755 ./portieris ${{targets.destdir}}/usr/bin/
+
+data:
+  - name: components
+    items:
+      # Only the keys matter
+      portieris:
```

Comment:
we should probalby standardize on something here.

I'd prefer a 'NOT_USED' value or something rather than a comment. thats due to having been editing several yamls with an program and the comments don't always get handled right.

Reply:
The melange [docs](https://github.com/chainguard-dev/melange/blob/main/docs/BUILD-FILE.md#data) are not clear on what the `data` block does.

Do we even need this block?

Reply:
With a bit of grepping it looks like a lot of packages have the block populated.